SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -99,6 +109,17 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         }
     }
     
+    @IBAction func muteSwitchToggled(_ sender: UISwitch) {
+        if let call = call {
+            call.isMuted = sender.isOn
+        } else {
+            NSLog("No active call to be muted")
```

Comment:
Does the switch still get toggled if the mute cannot be performed? If so I think it might mislead someone to think they will actually be muted or unmuted

Reply:
The sample code only shows the mute switch when the call is connected, so I think it should be fine.